buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -444,45 +439,46 @@ class BeamModulePlugin implements Plugin<Project> {
         commons_math3                               : "org.apache.commons:commons-math3:3.6.1",
         error_prone_annotations                     : "com.google.errorprone:error_prone_annotations:2.0.15",
-        gax                                         : "com.google.api:gax:$gax_version",
-        gax_grpc                                    : "com.google.api:gax-grpc:$gax_version",
+        gax                                         : "com.google.api:gax",
+        gax_grpc                                    : "com.google.api:gax-grpc",
         google_api_client                           : "com.google.api-client:google-api-client:$google_clients_version",
         google_api_client_jackson2                  : "com.google.api-client:google-api-client-jackson2:$google_clients_version",
         google_api_client_java6                     : "com.google.api-client:google-api-client-java6:$google_clients_version",
-        google_api_common                           : "com.google.api:api-common:1.8.1",
+        google_api_common                           : "com.google.api:api-common",
         google_api_services_bigquery                : "com.google.apis:google-api-services-bigquery:v2-rev20191211-$google_clients_version",
         google_api_services_clouddebugger           : "com.google.apis:google-api-services-clouddebugger:v2-rev20191003-$google_clients_version",
         google_api_services_cloudresourcemanager    : "com.google.apis:google-api-services-cloudresourcemanager:v1-rev20191206-$google_clients_version",
         google_api_services_dataflow                : "com.google.apis:google-api-services-dataflow:v1b3-rev20190927-$google_clients_version",
         google_api_services_pubsub                  : "com.google.apis:google-api-services-pubsub:v1-rev20191111-$google_clients_version",
         google_api_services_storage                 : "com.google.apis:google-api-services-storage:v1-rev20191011-$google_clients_version",
-        google_auth_library_credentials             : "com.google.auth:google-auth-library-credentials:$google_auth_version",
-        google_auth_library_oauth2_http             : "com.google.auth:google-auth-library-oauth2-http:$google_auth_version",
-        google_cloud_bigquery                       : "com.google.cloud:google-cloud-bigquery:1.108.0",
-        google_cloud_bigquery_storage               : "com.google.cloud:google-cloud-bigquerystorage:0.125.0-beta",
+        google_auth_library_credentials             : "com.google.auth:google-auth-library-credentials",
+        google_auth_library_oauth2_http             : "com.google.auth:google-auth-library-oauth2-http",
+        google_cloud_bigquery                       : "com.google.cloud:google-cloud-bigquery",
+        google_cloud_bigquery_storage               : "com.google.cloud:google-cloud-bigquerystorage",
         google_cloud_bigtable_client_core           : "com.google.cloud.bigtable:bigtable-client-core:1.13.0",
```

Comment:
This artifact is not in GCP Libraries BOM.

Reply:
Can missing GCP libs be added?

Reply:
As of now we don't have plan to increase the BOM coverage to keep maintenance cost low. Low maintenance burden leads frequent release cycle. When we find the burden is still low, we may add them.

Reply:
Have we confirmed that GCS artifacts not included in this BOM do not conflict with artifacts included in the BOM ?

Reply:
Which library are you referring to?

There’s no additional conflicts. Linkage Checker verified that.

Reply:
GCP libraries you already commented about that are not included in the BOM. For example " google_cloud_bigtable_client_core". I guess it should be fine if linkage checker passes. Note that our existing integration tests are not that useful here since they mostly check one GCS service at a time while users may use a combination.

Reply:
Yes, Linkage Checker detects the discrepancy if there's any.

> Note that our existing integration tests are not that useful here since they mostly check one GCP service at a time while users may use a combination.

This change is not intended to fix compatibility with users' libraries. It's about how Beam maintains the versions of these Google libraries with ease.